etna/transforms/math/apply_lambda.py

```diff
@@ -0,0 +1,163 @@
+import warnings
+from typing import Callable
+from typing import Optional
+
+import pandas as pd
+
+from etna.transforms import PerSegmentWrapper
+from etna.transforms import Transform
+from etna.transforms.utils import match_target_quantiles
+
+
+class _OneSegmentLambdaTransform(Transform):
+    """Instance of this class applies input function transformation to one segment data."""
+
+    def __init__(
+        self,
+        in_column: str,
+        inplace: bool,
+        out_column: Optional[str],
+        transform_func: Callable[[pd.DataFrame], pd.DataFrame],
+        inverse_transform_func: Optional[Callable[[pd.DataFrame], pd.DataFrame]] = None,
+    ):
+        """
+        Init OneSegmentLambdaTransform.
+
+        Parameters
+        ----------
+        in_column:
+            column to apply transform
+        out_column:
+            name of added column. If not given, use ``self.__repr__()``
+        transform_func:
+            function to transform data
+        inverse_transform_func:
+            inverse function of transform_func
+        inplace:
+            * if True, apply logarithm transformation inplace to in_column,
+            * if False, add column add transformed column to dataset
+        Warnings
+        --------
+        throws if inplace=True and out_column is initialized, transformation will be applied inplace
```

Comment:
There is problem with rendering: something like `inplace=False`
<img width="813" alt="image" src="https://user-images.githubusercontent.com/33594071/175346463-69cd91cd-77da-4e5a-9cb5-7eb304249f49.png">